src/router/index.js

```diff
@@ -41,18 +41,18 @@ export default {
     overView: '/home/wallet/overview',
   },
   registries: {
-    overview: '/home/registries',
+    overview: '/home/registries/home',
```

Comment:
double home looks weird
```suggestion
    overview: '/home/registries/overview',
```